internal/upload/upload.go

```diff
@@ -268,9 +291,7 @@ func NewHandler(
 
 		validator.Validate(vr)
 
-		upload := uploadData{Account: vr.Account}
-		response := responseBody{RequestID: vr.RequestID, Upload: upload}
-		jsonBody, err := json.Marshal(response)
+		jsonBody, err := createUploadRequest(vr)
```

Comment:
I like this.  To me, this is a great example of lessening the cognitive load.  In most instances, I probably don't care about how the upload response is getting created.  If I do care about how the upload response is getting created, then I can jump into that function and check it out.

:+1: